docs/en/internationalization.md

```diff
@@ -0,0 +1,132 @@
+
+* `admin.php` for anything related to FreshRSS administration
+* `conf.php` for configuration
+* `feedback.php` contains translations of feedback messages
+* `gen.php` stores what is global to FreshRSS (`gen` stands for “general”)
+* `index.php` for the main page that lists feeds and the About page
+* `install.php` contains strings related to the installation
+* `sub.php` for subscription management (`sub` stands for “subscription”)
+* `user.php` contains some strings related to the User model
+
+This organization makes it possible to avoid a single huge translation file.
+
+The translation files are quite simple: it's only a matter of returning a PHP array containing the translations. As an example, here's an extract from [`app/i18n/fr/gen.php`](/app/i18n/fr/gen.php):
+
+```php
+<?php
+
+return array(
+	'action' => [
+		'actualize' => 'Actualiser',
+		'back_to_rss_feeds' => '← Retour à vos flux RSS',
+		'cancel' => 'Annuler',
+		'create' => 'Créer',
+		'disable' => 'Désactiver',
+	),
+	'freshrss' => array(
+		'_' => 'FreshRSS',
+		'about' => 'À propos de FreshRSS',
+	),
+	// ...
+];
+```
+
+Each value can be referenced by a key: it consists of a series of identifiers separated by dots. The first identifier indicates from which file to extract the translation, while the following ones indicate array entries. Thus, the `gen.freshrss.about` key is referencing the `about` entry from the `freshrss` entry which is part of the main array returned by the `gen.php` file. This allows us to further organize our translation files.
+
+You should not have to write the array by yourself and we provide several commands to ease the manipulation of these files. Let's see some common use cases.
+
+## Add support for a new language
+
+If you want to add support for your own language which is not supported by FreshRSS yet, you can call this command:
+
+```console
+$ make i18n-add-language lang=[your language code]
+```
+
+You must replace `[your language code]` by the language tag of your language. It must follow the [IETF BCP 47 standard](https://en.wikipedia.org/wiki/IETF_language_tag). For instance, English is `en` and French is `fr`. You can target a specific region with a subtag, for instance `pt-br` for Brazilian Portuguese. If your not sure of the code, Wikipedia might be a good start to find it or you can ask us for help too.
+
+The command will create a new subfolder under `app/i18n/` and copy the strings from the reference language (i.e. English). It will also mark all the translations with a special tag represented by a comment: `// TODO - Translation`. We'll see in the next section how to translate the strings.
+
+## Translate the interface
+
+You might have noticed some strings are not translated in the interface. These strings are usually in English while you chose a different language. This is because we mostly speak English or French and it's pretty difficult to us to speak all the different languages!
```

Comment:
```suggestion
You might have noticed some strings are not translated in the interface. These strings are usually in English if you've selected a different language. This is because we mostly speak English or French and it's pretty difficult to us to speak all the different languages!
```

Reply:
I think this change has not the same sense than my initial sentence. Maybe `whereas` instead of `if`?

Reply:
```suggestion
You might have noticed some strings are not yet translated from English even though you've selected a different language. This is because we mostly speak English or French and it's pretty difficult to us to speak all the different languages!
```